main/ahb_for_sles.go

```diff
@@ -666,3 +671,9 @@ func RunShellCommand(timeout time.Duration, name string, args ...string) (string
 
 	return outb.String(), nil
 }
+
+func isSupportedOS() bool {
+	catCommand := "cat /etc/os-release | grep ^NAME="
+	output, _ := RunShellCommand(0, "bash", "-c", catCommand)
+	return strings.Contains(strings.ToLower(string(output)), "sles")
```

Comment:
This breaks for HPC as it has `sle`. Also this is somewhat dangerous as the user could have tempered with `/etc/os-release`. If we are only using one file for verification we should check the integrity of the file with the rpm database.